src/info/langs/mod.rs

```diff
@@ -90,23 +88,6 @@ fn filter_languages_on_type(types: &[LanguageType]) -> Vec<tokei::LanguageType>
         .collect()
 }
 
-fn get_ignored_directories(user_ignored_directories: &[PathBuf]) -> Vec<String> {
-    let mut ignored_directories = Vec::new();
-    if !user_ignored_directories.is_empty() {
-        let re = Regex::new(r"((.*)+/)+(.*)").unwrap();
-        for user_ignored_directory in user_ignored_directories {
-            let dir = user_ignored_directory.display().to_string();
-            if re.is_match(&dir) {
-                let prefix = if dir.starts_with('/') { "**" } else { "**/" };
-                ignored_directories.push(format!("{prefix}{dir}"));
-            } else {
-                ignored_directories.push(dir);
-            }
-        }
-    }
-    ignored_directories
-}
```

Comment:
Was meant to resolve this issue https://github.com/XAMPPRocky/tokei/issues/564.

But it turns out that [gitoxide's work_dir function returns a relative path ](https://github.com/Byron/gitoxide/blob/9fbed4b97cd31785d11cbc98c44a8332776a847c/gix/src/repository/location.rs#L44)